### ISSUE TITLE
No import notepad option

### ISSUE THREAD

**Issue by BrennanAK** [mvysny/aedict#871]
Hi

Been using your app for ages and even subscribed as it's helped me so much with my studies. 

However, I've recently gotten a new phone and much to be displeasure and shock, I can't find a way to transfer my notepad entries over. 

That's thousands of entries I had categorized for quizzes that I can no longer use.

Please, I would like to know what to do about this as there is information on your website, an export feature (which only seems to be for ankidroid), but no clear way to follow any instructions.

Ideas?

**Comment by mvysny** [comment]
Hi, thank you so much for your kind support!

The only way to transfer the files is by synchronizing them via aedict online. Please read more here: https://mvysny.github.io/aedict-blog/Aedict-Sync/
Just enable syncing on both phones and you should be ready to go.

**Comment by BrennanAK** [comment]
Hi,

Thanks very much for your quick reply here.

I've just been onto aedict online, went through all the settings and pages
but couldn't find "sync" anywhere, nor can I see it anywhere on the app.

I'm using Version 3.50.13 and am subscribed to aedict ultimate.

Would you mind terribly walking me through this?

Thanks,

Brennan Kenefick

**Comment by BrennanAK** [comment]
PS, on Android by the way

**Comment by mvysny** [comment]
It may be because you're using file based storage. Please make sure you
have the umn storage activated.

**Comment by BrennanAK** [comment]
This worked!!!

The instructions on your blog helped immensely. Couldn't find Aedict sync
before on user settings so I was really confused, but sure enough it showed
up after going into expert settings and changing the file base system.

Was further confused as I was focused on looking in the "export" settings
and it had the option there saying MNU storage had been selected. I mistook
this to mean I was already using that as my database type.

In any case, thank you for writing and pointing me towards that blog entry
which solved the problem.

All my flashcards have been successfully migrated and I can stop worrying.

Keep up the great work on the app!

All the best,

Brennan Kenefick

**Comment by mvysny** [comment]
Thank you kindly, I'm happy that the issue got resolved. It seems that I could have done a better job documenting this, since it's confusing. Hopefully this thread will help clearing out this issue. Best regards,
Martin :)

**Comment by BrennanAK** [comment]
Hi Martin,

Was hoping you could help me with two issues I've noticed with the
dictionary lately.

(Note: Galaxy S10+, Android version 9, pie)

Issue 1, In the kanji tab, pinyin is no longer displayed correctly. This is
a bit inconvenient as it was REALLY nice being able to just see the
Mandarin pronunciation of the character in aedict instead of having to go
into pleco to see it.

Issue 2, in the kanji search by radicals, many radicals seem to have their
icons missing. They can still be selected and still bring up the correct
filters but it's impossible to know which one is which.

I've attached pictures to illustrate my meaning.

Please let me know what you think.

I'm currently on Aedict 3.50.15 which I believe is the latest version, at
least on Google play.

All the best,

Brennan

**Comment by mvysny** [comment]
Hi Brennan, looks like some kind of font issue. I can't find the attachments. Could you please open 2 separate bug reports regarding those?

**Comment by BrennanAK** [comment]
Hi Martin,

Thank you so much for the quick reply.

It turned out that the custom font I was using was the issue as you
suggested.

Selecting "do not use Japanese font" in expert settings fixed the issue.

Unfortunate that that particular font couldn't be used but everything is
back to normal.

Thank you again for the reply and the help.

All the best,

Brennan

**Comment by BrennanAK** [comment]
Hello,

I'd like to ask when you expect a working update to be released for this
app.

Since the latest Android update it's stopped reading the clipboard and thus
stopped working completely.

I love your apps, your work and want to continue to use and support them
but I can't justify paying for something that doesn't work.

Thanks!

Brennan Kenefick

**Comment by mvysny** [comment]
Hi, I'm really sorry for the delay. I assume you're referring to https://github.com/mvysny/aedict/issues/889 . In order to develop the support for Android 10 properly, it's best to actually own an Android 10 device. My Xiaomi Mi A2 Lite is part of Android One and has been promised at least two major updates (which include Android 10). Just recently Xiaomi promised to also release Android 10 for Mi A2 Lite. There are hints that the upgrade is under testing, so it might be out this month (fingers crossed). However, I'll try Android Emulator for Android 10 again - I remember that I had trouble reproducing the issue in the Emulator, but perhaps those have been fixed. Let me try again.

**Comment by mvysny** [comment]
Please let's continue the discussion at #889

**Comment by BrennanAK** [comment]
Thanks for the advice on that thread (apologies didn't realize that there
was already a bug report thread open on this).

The workaround works for most apps, not for WeChat though, which is where I
find myself needing this the most. Options to open/copy with another app
don't show up there.

It is almost as fast though to keep the dictionary open and paste directly
unfortunately.

Guess we need to cross our fingers this won't be a problem with the next
Android update!

Thanks for your quick replies.
